connect/runtime/src/main/java/org/apache/kafka/connect/runtime/AbstractStatus.java

```diff
@@ -24,8 +24,9 @@ public enum State {
         UNASSIGNED,
         RUNNING,
         PAUSED,
+        STOPPED, // Only ever visible to users for Connector instances; never for Task instances
```

Comment:
Could someone be relying on the integer values of this enum? Maybe it's better to append the new state to the end.

Reply:
This class isn't part of the public API and I don't believe the ordinal values for it are used anywhere internally, but it doesn't hurt to make this change just in case. Good call, done 👍